ibm_catalog.json

```diff
@@ -0,0 +1,184 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-scc",
+        "label": "Cloud automation for Security and Compliance Center Workload Protection",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "security"
+        ],
+        "keywords": [
+          "SCC",
+          "security",
+          "compliance",
+          "workload-protection",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution"
+        ],
+        "short_description": "Creates and configures IBM Security and Compliance Center resources",
+        "long_description": "This architecture supports creating and configuring IBM Security and Compliance Center resources.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-scc-workload-protection/blob/main/solutions/instances/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-scc-workload-protection/main/images/scc-icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Security and Compliance Center Workload Protection",
+            "description": "Creates and configures a Security and Compliance Center Workload Protection instance."
+          }
+        ],
+        "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues](https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+        "flavors": [
+          {
+            "label": "Workload Protection",
+            "name": "workload-protection",
```

Comment:
I'm trying to close on the exact name of the variation. It will be something like `Full flexibility` but Ill try confirm that for you asap